docs/farming-&-staking/farming/space-acres/_category_.json

```diff
@@ -1,8 +1,5 @@
 {
-    "label": "Space Acres (Recommended)",
-    "position": 2,
-    "link": {
-      "type": "generated-index",
-      "description": "Autonomys GUI Application Space Acres"
-    }
+    "label": "✨ Space Acres",
+    "position": 3,
```

Comment:
Similarly inside the farming category - I'd suggest keeping the order: Getting Started, Space Acres, Advanced CLI, Common Problems, Additional Guides